plugins/github/plugin.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "GitHub",
+  "description": "Adds a GitHub authentication provider, support for link unfurling.",
```

```suggestion
  "description": "Adds a GitHub integration for link unfurling.",
```

I want to remove "auth provider" type language, otherwise people will think you can login with it